test/topics_test.rb

```diff
@@ -3,6 +3,43 @@
 describe "topics" do
   topics.each do |topic|
     describe "#{topic} topic" do
+      it "has a valid name" do
+        assert valid_topic?(topic), invalid_topic_message(topic)
+      end
+
+      it "has valid aliases" do
+        aliases = aliases_for(topic)
+
+        aliases.each do |topic_alias|
+          assert valid_topic?(topic_alias), invalid_topic_message(topic_alias)
+          refute_equal topic_alias, topic,
+                       "alias '#{topic_alias}' must not be the same as the topic"
```

Comment:
I'm annoying but I'd be tempted to say loop through the `aliases` as separate tests or some other way to have a single `refute` otherwise you need to fix a test fail to see the next failure etc. This is generally an argument to me for one `refute` or `assert` per test but that may be totally overkill so: feel free to ignore my personal preference 😆.

Reply:
My assumption is most of the time the aliases and related topics lists will be short. So I think the error message that will be shown will be clear enough.

Reply:
Is there an internal max for aliases we should enforce?

Reply:
Hmm, good point Jeff. Maybe 10? I'll add another test.